### PR TITLE
feat: Bybit Spot

### PR DESCRIPTION
**Pull request by villekuosmanen**

### CODE REVIEW

File: cryptofeed/exchange/bybit_spot.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+                ]
+            }
+        }
+        """
+        pair = self.exchange_symbol_to_std_symbol(msg['params']['symbol'])
+        data = msg['data']
+        delta = {BID: [], ASK: []}
+
+        # TODO are all updates snapshots, or only the first one?
+        # API doesn't tell. This code assumes all are snapshots
```
Comment (tristan-murfitt-elw) [inline cryptofeed/exchange/bybit_spot.py]: need to check this

**Review by tristan-murfitt-elw — COMMENTED** [review]
all good but need to check the Book updates

**Comment by villekuosmanen** [comment]
Every book update seems to have 40 bids and asks so I presume they are all snapshots.

I noticed there was a bug when subscribing to multiple symbols, fixed that as well. This should now be ready.

**Comment by tristan-murfitt-elw** [comment]
- Added deltas because this was skipping to publish every snap (as we only publish a snap every 30s to save bandwidth)
- Use the same symbol normalisation as elsewhere
- Fixed tick callback not converting to Decimal


```
python _build_live_orderbook.py 

Waiting for first OB snap to arrive (max 30s)...
-------
Got new OrderBook snapshot at 2022-01-04T17:32:49.477017:
New snap:
b = 46817.1400|0.04, 46817.1300|0.11, 46812.7700|0.07, 46810.5300|0.03, 46809.9300|0.00 
a = 46820.2100|0.11, 46825.8000|0.32, 46826.0900|0.00, 46826.1800|0.03, 46833.3800|0.21
OB snapshot 40|40 took 0.001s, 0 updates since last snap
-------
-------
Got new OrderBook snapshot at 2022-01-04T17:33:21.353220:
Build from updates == New snapshot! :)
New snap:
b = 46809.8700|0.11, 46805.3400|0.08, 46805.2600|0.00, 46805.2500|0.04, 46803.6500|0.02 
a = 46812.2300|0.11, 46812.8800|0.01, 46818.4000|0.07, 46821.5200|0.02, 46823.5500|0.21
OB snapshot 40|40 took 0.001s, 100 updates since last snap
-------
```

**Review by tristan-murfitt-elw — APPROVED** [review]

**Pull request merged**